### DISCUSSION TITLE
Ride vehicles research category

### DISCUSSION THREAD

**Discussion by ZeeMaji** [OpenRCT2/OpenRCT2#17989] [Feature requests]
Something that i've always thought would be a nice addition is a new research category that'd be solely for additional vehicles for already researched ride types.
This would be similar to the ride improvements research category from RCT1 but would solely be for additional vehicles.
I think it would make for a nice quality of life improvement and make research more accurate in imported RCT1 scenarios as right now due to new vehicles being lumped in with other categories you usually have to research a ton of vehicle variants before you start researching new coaster types (eg: have to research all of the classic mini coaster trains before you can get a new coaster type)

**Comment by sjoerddebruin** [comment]
This would also benefit parks like Leafy Lake that have tons of boat hire vehicles in the invention list.
